### PR TITLE
Add support for python2.7

### PR DESCRIPTION
**Pull request by fabiocerqueira**
Python 2.7 won't be officially supported next year(2020), but it will still alive in thousands of companies. This PR allows depx parser python2.7 and python3 code

### CODE REVIEW

**Comment by anapaulagomes** [comment]
@tilboerner would you do us the honors of this review?

File: tests/test_parsing.py [hunk 1]
```diff
@@ -179,6 +181,10 @@ def test_find_imports_from_text__async_def():
     (os.getcwd() + '/tests/fake_project/another_fake_module', True),
     (os.getcwd() + '/tests/fake_project/fake_module', True),
     (os.getcwd() + '/tests/fake_project/random_folder', False),
+    # Python 2.7
+    (os.getcwd() + '/tests/project_py27/another_fake_module', True),
+    (os.getcwd() + '/tests/project_py27/fake_module', True),
+    (os.getcwd() + '/tests/project_py27/random_folder', False),
```
Comment (anapaulagomes) [inline tests/test_parsing.py]: Would be nice to have the following folder structure:

```
tests/
   fake_project/
       py27/
       py3/
```

Seems a bit weird having `project_py27` and `fake_project` at the same level (at first glance isn't possible to say the difference between them).

**Comment by tilboerner** [comment]
@anapaulagomes I'll be happy to review, but I won't be able to until next week when I'm back from vacation. :)

**Comment by anapaulagomes** [comment]
Take your time (and enjoy your vacation). 🍸 @tilboerner

**Comment by fabiocerqueira** [comment]
@anapaulagomes depx still requires Python 3.4+ to run but you can inspect/parser Python 2 code with it, too. So, there is no need to add py27 to the tox.ini

File: tests/test_parsing.py (hunk 1 above)
Comment (fabiocerqueira) [reply tests/test_parsing.py]: Ok, I'll do it :D

File: tests/test_parsing.py [hunk 2]
```diff
@@ -274,3 +299,11 @@ def test_filter_top_level_names():
         'to_name': '',
     }
     assert list(filter_top_level_names([dependency])) == [expected]
+
+
+@pytest.mark.parametrize('src,module_version', [
+    ('print "python27", 1000L', ModuleAST27),
+    ('print("python3")', ModuleAST3),
```
Comment (tilboerner) [inline tests/test_parsing.py]: If there's really a need for differentiating between Python 3 minor versions in `_parse_ast`, we should test for them.

**Comment by fabiocerqueira** [comment]
@tilboerner I'll have a look in your comments, but I'll answer or apply the change only next week. Thanks for the review :D

**Comment by tilboerner** [comment]
@fabiocerqueira @anapaulagomes I've been thinking about this some more, and now I believe it would be best to have a command-line switch to force `--py2` parsing, and use the builtin `ast` otherwise. Here are my reasons:

- People already have a Python 3 interpreter to run their code with; it's natural to use it for parsing.

- Adding more parsing options complicates the system with little gain beyond Python 2.

- *Not* using the same interpreter (and virtualenv) that is going to run the code closes off some easy options to find out more about the imported dependencies, like finding and categorizing them. We never made an explicit decision about this, and I'd rather not cement a decision as a side-effect of helping Python 2 live a little longer. :)

- We can explicitly make the `--py2` mode a temporary helper, that might not provide full functionality, and that will get dropped at some point in the future, allowing us more flexibility.

What do you guys think?

**Comment by tilboerner** [comment]
Stale PR, closing.

**Pull request closed without merging**